realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/CredentialsTests.kt

```diff
@@ -239,6 +239,7 @@ class CredentialsTests {
                 }
             }
         }
+        app.close()
```

Comment:
A bit weird to have this mixed handling of `app.close`. Explicit here but also in `tearDown`. I guess `app` could be a local variable just for this test.

Reply:
Yeah, this is a leftover from some things I was trying when adding the obfuscation. Forgot to clean up.